src/views/jsonImportExport.vue

```diff
@@ -0,0 +1,231 @@
+<template>
+  <ion-page>
+    <ion-header>
+      <ion-toolbar>
+        <ion-title>JSON Editor</ion-title>
+        <ion-buttons slot="end">
+      <ion-button @click="exportJson()" :disabled = isDisabled >
```

Comment:
Instead of using isDisabled create a method called isJsonUpdated returning a boolean value.